src/templates/components/PaidForHeader.svelte

```diff
@@ -22,10 +18,7 @@
 	<div class="{templateType}-paid-content-and-title">
 		<div class="paid {templateType}-paid-content">
 			<strong>Paid content</strong>
-			<Button
-				aria-controls="popup"
```

Comment:
Why are we losing `aria-controls` and `aria-expanded`?

Reply:
redundant as they're already on the `Button` component! https://github.com/guardian/commercial-templates/blob/jlk/upgrade-sveltekit/src/templates/components/Button.svelte#L1